src/makers/cotx/hotspots.ts

```diff
@@ -0,0 +1,91 @@
+import HotspotIcon from './cotx.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const COTX_V1 = {
+  name: 'COTX Gateways',
+  onboardType: 'BLE',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Helium to identify issues with your Hotspot in a secure way.</white></b>\n\nHelium will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@helium.com</b></purple> from the email used to purchase the Hotspot.',
+      power: [
+        'Plug in the provided power adapter into an outlet near a window.',
+        'The COTX Gateway Miner will show a red LED light once it’s powered on.',
+      ],
+      bluetooth: [
+        'There is no pairing button on the COTX Gateway Miner.',
+        'Bluetooth is automatically enabled for 5 minutes after the COTX Gateway Miner is powered on.\n\nHotspot can take up to 1 minute to fully boot up.',
+      ],
+    },
+    ja: {
+      diagnostic:
+        '<b><white>診断サポートにより、HeliumはHotspotの問題を安全な方法で特定できます。</white></b>\n\nHeliumが秘密キーにアクセスすることはありません。お使いのHotspotにのみアクセスし、ネットワーク上の他のデバイスにはアクセスしません。\n\n診断サポートをオプトアウトする場合は、Hotspotの購入時に使用したメールアドレスを使用して、<purple><b>support@helium.com</b></purple>までメールでご連絡ください。',
```

Comment:
same thing here. Even though it's in Japanese, you can easily replace Helium and the Helium support email with your own company name and support email.

Reply:
> same thing here. Even though it's in Japanese, you can easily replace Helium and the Helium support email with your own company name and support email.

Ok, I've updated this questions.

Reply:
> same thing here. Even though it's in Japanese, you can easily replace Helium and the Helium support email with your own company name and support email.

Hi, I have submitted the code again. Please review it when you have time.